api/base/filters.py

```diff
@@ -148,6 +148,19 @@ def _parse_date_param(self, field, field_name, op, value):
                 'value': stop
             }]
 
+    def bulk_get_values(self, value, field):
+        """
+        Returns list of values from query_param for IN query
+
+        If url contained `/nodes/?filter[id]=12345, abcde`, the returned values would be:
+        [u'12345', u'abcde']
+        """
+        if value[0] == '[' and value[-1] == ']':
```

Comment:
Superfluous `if` statement, `lstrip` and `rstrip` will do nothing if the characters don't match.

```python
In [7]: s
Out[7]: '[abcde,fghij,klmno]'

In [8]: s = s.lstrip('[').rstrip(']')

In [9]: s
Out[9]: 'abcde,fghij,klmno'

In [10]: s = s.lstrip('[').rstrip(']')

In [11]: s
Out[11]: 'abcde,fghij,klmno'
```